### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by kdvalin**
# So Far
- Updated `react-scripts` and `create-react-app` to v4
- Updated `three` to be on r126.
- Updated `aframe` to be on v1
- Other small dependency updates

# Currently Broken
- ~~Light indicators~~

# Things to do before merging
- [x] Update the Dockerfile to use 12.18
- [x] Update dependencies on backend as well
- [ ] Test on dev.learnmyr.org
- [ ] Update documentation for future developers

### CODE REVIEW

File: src/utils/AFramePhysics.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ AFRAME.registerComponent("force-pushable", {
 //This set aframe entity to different layer (range from 0-31)
 //all the regular MYR entities will goes to layer 0
 //And other, such as grid, light indicator will goes to layer 1 so it won't take effect of user created light
-AFRAME.registerComponent("layer",{
```
Comment (kdvalin) [inline src/utils/AFramePhysics.js:30]: Renamed this line from `layer` to `lightlayer` since `layer` was already used in a new AFrame version

Comment (kdvalin) [reply src/utils/AFramePhysics.js:30]: Potential area that breaks light indicators

File: src/utils/AframeRegIndicator.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ AFRAME.registerComponent("spotlightindicator",{
         const data = this.data;
 
         /*      define geometry     */
-        let cone = new THREE.CylinderGeometry(.1,.75,1,24,1,true);
```
Comment (kdvalin) [inline src/utils/AframeRegIndicator.js:19]: Updated *Geometry to *BufferGeometry since THREE.Geometry was removed in a subsequent release.

Comment (kdvalin) [reply src/utils/AframeRegIndicator.js:19]: Another potential area that breaks light indicators

**Review by samuel-zuk — DISMISSED** [review]

**Review by Jlu18 — DISMISSED** [review]

**Pull request merged**